ChineseVersionDocs/README.cn.md

```diff
@@ -0,0 +1,47 @@
+![](https://user-images.githubusercontent.com/176499/96893278-ebc67580-1460-11eb-9530-d5df3a3d65d0.png) [![NPM Package](https://img.shields.io/npm/v/hardhat.svg?style=flat-square)](https://www.npmjs.org/package/hardhat) [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/NomicFoundation/hardhat/badge)](https://www.gitpoap.io/gh/NomicFoundation/hardhat)
```

Comment:
I think we should use a slightly different directory structure. Something like `/translations/chinese/hardhat-core/README.md` and `/translations/chinese/docs/README.md` maybe?

Reply:
That will be a much better directory structure. I will work on it soon.